mtz-wizard-step-errors-behavior.html

```diff
@@ -0,0 +1,94 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="mtz-wizard-step-behavior.html">
+
+<script>
+  window.mtz = window.mtz || {};
+
+  /**
+   * `mtz.WizardStepErrorsBehavior`
+   * Extends the mtz.WizardStepBehavior to allow setting an errors array that will perform a lookup for the form
+   *  elements associated with each error then set their error message and state.
+   *
+   * @polymerBehavior mtz.WizardStepErrorsBehavior
+   * @demo demo/mtz-wizard-step.html
+   */
+  mtz.WizardStepErrorsBehavior = [mtz.WizardStepBehavior, {
+    properties: {
+      /* The attribute to assign the error-message to */
+      errorMessageAttribute: {
+        type: String,
+        value: 'error-message'
+      },
+      /**
+       * The list of all errors to associate with the form
+       * @type {Object.<{step: String, field: String, code: String, message: String}>}
```

Comment:
`{step: String, field: String, code: String, message: String}` should be the type.

I prefer using `typedef` with `property` instead of an anonymous record row.